### PR TITLE
Fix too many open files issue (#3630)

### LINKED ISSUE #3630: Too many open files error
The issue is related to neo-project/neo#413

- Version: v1.3.8
- Platform: Ubuntu 16.04-x64
- Reproduction: Try to connection a IP which cannot be connected via TCP repeatedly, the error will be occurred later

### PR DESCRIPTION
**Pull request by yongjiema**
Fix #3630

### CODE REVIEW

File: src/core/Akka/IO/TcpConnection.cs [hunk 1]
```diff
@@ -618,6 +618,7 @@ private bool HasStatus(ConnectionStatus connectionStatus)
         protected override void PostStop()
         {
             if (Socket.Connected) Abort();
+            else CloseSocket();
```
Comment (Aaronontheweb) [inline src/core/Akka/IO/TcpConnection.cs]: Nice catch

**Comment by Aaronontheweb** [comment]
Looks good to me - thanks for your contribution @yongjiema

File: src/core/Akka/IO/TcpConnection.cs (hunk 1 above)
Comment (yongjiema) [reply src/core/Akka/IO/TcpConnection.cs]: My pleasure

**Comment by yongjiema** [comment]
> Looks good to me - thanks for your contribution @yongjiema

My pleasure

**Comment by vncoelho** [comment]
Congratulations @yongjiema for this cross-platform contribution, Neo will be thankful to this :dagger:

**Pull request merged**